app/app.go

```diff
@@ -185,13 +185,13 @@ func (a *App) metricsReporterInterceptor(
 	reportedFailures := false
 	res, err := handler(ctx, req)
 	if err != nil {
-		l.WithError(err).Error("error processing request")
+		l.WithError(err).Error(err.Error())
```

Comment:
I think we can keep "error processing request" here. err & err.Error() will probably result in the same thing showing twice in the log.